Use unordered list element for references

metters opened the pull request:
Uses an unordered list to display several references, instead of buttons. Resolves GH-947

### Before
<img width="864" alt="Screenshot 2025-03-05 at 21 47 21" src="https://github.com/user-attachments/assets/b7c52a56-efda-46fc-8846-b0bb6a007acb" />

### After
<img width="500" alt="Screenshot 2025-03-06 at 20 57 57" src="https://github.com/user-attachments/assets/4def9413-0698-4d50-a113-60ded73a4f44" />
<img width="250" alt="Screenshot 2025-03-06 at 20 58 08" src="https://github.com/user-attachments/assets/fb0e374d-3c88-4b42-baa4-62b3d8659169" />

1Maxnet1 reviewed (approved):
Looks great, thanks! I'd wait for vkrauses anwser on the issue and merge it then :)

metters commented:
I added the links to the upcoming events FOSSGIS. The current result can be seen on the updates screenshots in the description.

In order to avoid another pull request, there are some points we should agree upon before merging:

1. All of the links lead to videos (in which the viewer can see the slides), so I removed it, because it seemed redundant
2. I added a flag emoji, depending on the language of the event, the link refers to. There are several reasons why this is better:
  * the `(in German)` did not look very good and it takes away space
  * if there are several links in German it is quite redundant, because we should add this to all of the links
  * What about the other links? Should they all have `(in English)` appended?
3. I unified the structure and naming:
  * I believe this list will become longer in the future, so it'll better to already group/order the presentations by year
  * The name of the conference is not abbreviated but fully spelled out (except for the last two ones)
5. Because there are two presentations on the same event, I added the title of the presentation, but only for those two.

Nothing is set in stone, just let me know and I'll edit the list until we have an appealing solution

metters commented:
Maybe `Chaos Communication Congress` should be `Chaos Communication Congress 38C3` or `Chaos Communication Congress (38C3)`...

jbruechert commented:
The directon looks very good, I think the flags are a good solution as well.

I think the use of abbreviations for conferences and meetups is unfortunately more complicated.

* In case of "Free and Open source Software Developers' European Meeting" I had no idea what that was and had to guess it's the full name of FOSDEM, so in that case I think the abbreviation is much better known (please comment if I'm wrong).

* For the Congress both is very well known so both is fine I guess.

* In case of the Open Transport Meetup I'd say the abbreviation is lesser known and the whole meetup is still relatively small, so the full name is more helpful to find out what it is.

jbruechert commented on website/content/_index.html:
```diff
@@ -45,5 +45,28 @@ <h2>Public Transport Coverage</h2>
+<h3>2025</h3>
+<ul>
+  <li>
+     🇬🇧 <a target="_blank" href="https://fosdem.org/2025/schedule/event/fosdem-2025-4105-gnome-maps-meets-transitous-meets-motis/" >Free & Open source Software Developers' European Meeting<i class="bi bi-box-arrow-right m-2"></i></a>
```
```suggestion
     🇬🇧 <a target="_blank" href="https://fosdem.org/2025/schedule/event/fosdem-2025-4105-gnome-maps-meets-transitous-meets-motis/" >FOSDEM<i class="bi bi-box-arrow-right m-2"></i></a>
```

jbruechert commented on website/content/_index.html:
```diff
@@ -45,5 +45,28 @@ <h2>Public Transport Coverage</h2>
+     🇬🇧 <a target="_blank" href="https://fosdem.org/2025/schedule/event/fosdem-2025-4105-gnome-maps-meets-transitous-meets-motis/" >Free & Open source Software Developers' European Meeting<i class="bi bi-box-arrow-right m-2"></i></a>
+  </li>
+  <li>
+     🇬🇧 <a target="_blank" href="https://meet.systect.de/playback/presentation/2.3/d941f8412ff350b73e9084e0fd1da86394aff088-1739990087983" >Open Transport Meetup Import pipeline<i class="bi bi-box-arrow-right m-2"></i></a>
```
```suggestion
     🇬🇧 <a target="_blank" href="https://meet.systect.de/playback/presentation/2.3/d941f8412ff350b73e9084e0fd1da86394aff088-1739990087983" >Transitous Import Pipeline - Open Transport Meetup<i class="bi bi-box-arrow-right m-2"></i></a>
```

jbruechert commented on website/content/_index.html:
```diff
@@ -45,5 +45,28 @@ <h2>Public Transport Coverage</h2>
+<h3>2024</h3>
+<ul>
+  <li>
+     🇬🇧 <a target="_blank" href="https://meet.systect.de/playback/presentation/2.3/d941f8412ff350b73e9084e0fd1da86394aff088-1708540821542" >Open Transport Meetup Initial<i class="bi bi-box-arrow-right m-2"></i></a>
```
```suggestion
     🇬🇧 <a target="_blank" href="https://meet.systect.de/playback/presentation/2.3/d941f8412ff350b73e9084e0fd1da86394aff088-1708540821542" >Transitous - Open Transport Meetup<i class="bi bi-box-arrow-right m-2"></i></a>
```

jbruechert commented:
I changed it to use the full talk title and the conference short name were that is better known.
If everyone is okay with this, it's ready to merge now from my side.

![image](https://github.com/user-attachments/assets/b9a996a4-08d6-4ec9-8a2b-3a305d2d2261)

metters commented:
Thanks for taking care of this!
